tests/integration/concrete/campaign/shared/claim/claim.tree

```diff
@@ -19,7 +19,7 @@ Claim_Integration_Test
                └── when amount valid
                   ├── when Merkle proof not valid
                   │  └── it should revert
-                  └── when Merkle proof valid
+                  └── when valid Merkle proof
```

Comment:
The whole tree uses `valid` and `not valid` at the end of sentences. But we can make an exception for Merkle proof. In that case, the above branch should be `when invalid Merkle proof`

Reply:
nevermind, I'll bring it back